.travis.yml

```diff
@@ -92,7 +93,13 @@ script:
     make test CTEST_OUTPUT_ON_FAILURE=1 &&
     make format && git diff --exit-code &&
     make doxygen &&
-    make install DESTDIR=${PWD}/install && rm -rf ${PWD}/install/usr/local && rmdir ${PWD}/install/usr &&
+    make install DESTDIR=${PWD}/install && rm -rf ${PWD}/install/$HOME/cabana && rmdir ${PWD}/install/$HOME &&
```

Comment:
Why do we have this line at all?

Reply:
I guess I am confused what this line does.

Reply:
This line tests if `make install` installs any files outside of `CMAKE_INSTALL_PREFIX`.

Reply:
paths do not look right

Reply:
I take it back no that I looked at the [doc](https://www.gnu.org/prep/standards/html_node/DESTDIR.html)

Reply:
Huh? When using `DESTDIR` everything gets prefixed with it value.